packages/tv/src/withTVAndroidManifest.ts

```diff
@@ -0,0 +1,130 @@
+import { ExpoConfig } from "expo/config";
+import {
+  AndroidConfig,
+  ConfigPlugin,
+  WarningAggregator,
+  withAndroidManifest,
+} from "expo/config-plugins";
+
+import { ConfigData } from "./types";
+import { isTVEnabled, showVerboseWarnings } from "./utils";
+
+const pkg = require("../package.json");
+
+const { getMainActivity } = AndroidConfig.Manifest;
+
+export const withTVAndroidManifest: ConfigPlugin<ConfigData> = (
+  config,
+  params = {}
+) => {
+  const isTV = isTVEnabled(params);
+  const verbose = showVerboseWarnings(params);
+
+  return withAndroidManifest(config, async (config) => {
+    if (isTV) {
```

Comment:
Bail out of the entire plugin if `isTV` isn't enabled, skip instantiating an empty `withAndroidManifest`